nginx/ssl/cert.crt

```diff
@@ -0,0 +1 @@
+TOTALGARBAGEPLACEHOLDERSHIZ
```

Comment:
I wouldn't check these files in and instead, add them to `.gitignore`. Nginx would fail when it cannot read these files anyway?

Reply:
this was more of a bootstrapping for the average user for guidance on placing the ssl files to line up with the nginx conf changes, it would fail yes but only if they enable it, and if they enable they should atleast understand that they need to replace the ssl files.

i can remove these files if you prefer, but then i think git will not checkout the ssl folder as it will be empty

Reply:
(or am i missing something and there is actually selfsigned ssl stuff as part of this image?)

Reply:
>this was more of a bootstrapping for the average user for guidance on placing the ssl files to line up with the nginx conf changes, it would fail yes but only if they enable it, and if they enable they should atleast understand that they need to replace the ssl files.

I think we can find better ways to signal this. They still need to know where to look for these files and then actually open them (people rarely open cert files in text editors I think) to see your message :)

>i can remove these files if you prefer, but then i think git will not checkout the ssl folder as it will be empty

If you think we should have the `ssl` folder, we can have an empty `.gitignore` file in it to keep it in Git.

Reply:
noted makes sense :) 

and resolved